<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dock_Grid</class>
 <widget class="QWidget" name="Dock_Grid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid</string>
  </property>
  <property name="windowIcon">
   <iconset theme="configure-grid"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QWidget" name="central_widget" native="true">
     <layout class="QGridLayout" name="gridLayout1">
      <item row="0" column="0">
       <widget class="QGroupBox" name="check_enable">
        <property name="title">
         <string>Enable Grid</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spin_size">
           <property name="suffix">
            <string>px</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Shape</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="combo_shape">
           <item>
            <property name="text">
             <string>Square</string>
            </property>
            <property name="icon">
             <iconset theme="grid-square"/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle 1</string>
            </property>
            <property name="icon">
             <iconset theme="grid-triangle-h"/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Triangle 2</string>
            </property>
            <property name="icon">
             <iconset theme="grid-triangle-v"/>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Origin</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spin_x">
              <property name="minimum">
               <double>-8192.000000000000000</double>
              </property>
              <property name="maximum">
               <double>8192.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>y</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spin_y">
              <property name="minimum">
               <double>-8192.000000000000000</double>
              </property>
              <property name="maximum">
               <double>8192.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="button_reset">
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="button_move">
              <property name="text">
               <string>Move...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
